src/main/java/com/fundation/logic/common/MetadataImageExtractor.java

```diff
@@ -60,39 +60,41 @@ public void readAll() {
 
     /**
      * This method read mime type.
-     *
-     * @return
+     * @return void
      */
     public static void mimeType(String mimeType) {
-        String imageMimeType = mimeType.substring(0,9);
-        if ((imageMimeType.contains("MIME Type"))) {
-            if (mimeType.contains("image")) {
-                searchMimeType = "image";
+        String mimetype2 = mimeType;
+        String audioMimeType = mimeType.substring(0, 4);
+        if ((audioMimeType.contains("MIME"))) {
+            if (mimetype2.contains("image")) {
+                int initIndex = mimetype2.indexOf(":");
+                int freeSpace = 2;
+                mimeType = mimetype2.substring(initIndex + freeSpace, initIndex + 7);
+                searchMimeType = mimeType;
+            } else {
+                searchMimeType = "All";
             }
         }
     }
 
     /**
      * This method returns metadata image height.
+     * @return void
      */
     public static void height(String height) {
         if ((height.contains("Image Height"))) {
             int initIndex = height.indexOf(":");
             int endIndex = height.length();
             int freeSpace = 2;
             searchHeight = height.substring(initIndex + freeSpace, endIndex);
+        } else {
+            searchHeight = "All";
         }
     }
 
-    /**
-     * This method returns height image.
-     */
-    public static String getHeight() {
-        return searchHeight;
-    }
-
     /**
      * This method returns metadata image Width.
+     * @return void
      */
     public static void width(String width) {
```

Comment:
Use descriptive method names as it is difficult to understand what the method does just for the name